config/consensus.go

```diff
@@ -815,6 +818,7 @@ func initConsensusProtocols() {
 	v24.MaxAppProgramLen = 1024
 	v24.MaxAppKeyLen = 64
 	v24.MaxAppBytesValueLen = 64
+	v24.MaxAppSumKeyValueLens = 128
```

Comment:
wrong place, needs to be in vFuture

Reply:
By putting it here, the code that checks the sum of the lengths does not need to be separately feature-flagged.  It can check that the sum is  <= this value, even in old txns, in old protocol versions. That is, it is just making explicit something that was always true: the sum <= 128.

Reply:
thanks for explaining, let me think if it can break something

Reply:
While the code above would work correctly, it's wouldn't be the easiest to follow. I agree with @pzbitskiy in that it would be cleaner to add it to `vFuture`. The feature-flagging doesn't have to be fancy though; using
```golang
//  on version v27 we added MaxAppSumKeyValueLens to allow (key+value) < 128 bytes instead of key/value being 64 bytes each.
if sum := len(key) + len(value.Bytes);  cb.proto.MaxAppSumKeyValueLens != 0 && sum > cb.proto.MaxAppSumKeyValueLens {
```
instead of 
```golang
if sum := len(key) + len(value.Bytes); sum > cb.proto.MaxAppSumKeyValueLens {
```

would do it.